src/test/java/com/fryrank/TestConstants.java

```diff
@@ -47,7 +48,8 @@ public class TestConstants {
         TEST_TITLE_1,
         TEST_BODY_1,
         null,
-        TEST_ACCOUNT_ID
+        TEST_ACCOUNT_ID,
+            null
```

Comment:
nit: spacing is off

Reply:
Done in [98ac0c5](https://github.com/NickPriv/FryRankBackend/pull/83/commits/98ac0c5d7f9c27e6e75a242c815fd2e47c9cbdb6).